### ISSUE TITLE
NET income is not changing when there is tax

### ISSUE THREAD

**Issue by azaricstefan** [bytefury/crater#228]
**Describe the bug**
I created an example where I added tax 20%.
Created 3 invoices and linked 3 payments.
Sum amount is 1000.
Net income on dashboard is same 1000.
Net income should be 1000 - 20% from 1000 = 800

Tax is per invoice level, not per item.

**Expected behavior**

Net income should be 800

**Screenshots**
![image](https://user-images.githubusercontent.com/9116773/84448961-5ca48800-ac4c-11ea-902b-0dac204667cc.png)

**Please complete the following information:**
- Crater version: 3.1.1
- PHP version: 7.4 (from docker)
- Database type and version: MariaDB (docker version)

**Optional info**
- OS:  MacOS
- Browser: Chrome

**Comment by mohitpanjwani** [comment]
Yes, that's correct! Net Income currently doesn't take tax into account. 

I'm thinking of combining Expenses & Taxes and rename the label to "Expenses & Taxes", as we don't have room for an extra Tax field. 

What do you think?

**Comment by azaricstefan** [comment]
Here we are talking about 2 things:

1. **Fixing of Net income** to take Tax into account. Which will fix Net Income values.
2. Regarding **combining of Expenses and Tax**, I don't think that it's convenient to have them combined, if someone would like to get Tax for some time frame he could go to reports and do such thing.

We could think about splitting main dashboard into 3 graphs.

First one (current one) could have Income & Expense values.
Second one could have Expense & Tax
Third drawing could be bar graph which will show expense categories and values of how much was spent on each category for a given time period.
Fourth drawing could be again bar graph which will show income/receipts which will show what customers contributed in a particular time frame to the income.
